finite-state-machine/src/main/java/com/iluwatar/finite/state/machine/Main.java

```diff
@@ -0,0 +1,33 @@
+package com.iluwatar.finite.state.machine;
+
+/**
+ * Main.
+ */
```

Comment:
Here above the main class, please describe the pattern briefly and explain how the example fulfills it